### ISSUE TITLE
Incremental compilation fatally broken

### ISSUE THREAD

**Issue by AnHeuermann** [ThummeTo/FMIImport.jl#43]
When pre-compiling my package I get warnings that method definitions are overwritten:

```julia
WARNING: Method definition fmi2GetStartValue(FMICore.FMU2Component, Union{Nothing, Int64, UInt32, AbstractArray{Int64, 1}, AbstractArray{String, 1}, AbstractArray{UInt32, 1}, String, Symbol}) in module FMIImport at /path/to/FMIImport/snZaf/src/FMI2_int.jl:706 overwritten at path/to/FMIImport/snZaf/src/FMI2_ext.jl:708.
  ** incremental compilation may be fatally broken for this module **
```

And indeed there are two definitions for `fmi2GetStartValue`: https://github.com/ThummeTo/FMIImport.jl/blob/main/src/FMI2_ext.jl#L690 and https://github.com/ThummeTo/FMIImport.jl/blob/main/src/FMI2_int.jl#L706.

Same issue for `fmi2GetNamesAndInitials`
```julia
WARNING: Method definition fmi2GetNamesAndInitials(FMICore.fmi2ModelDescription) in module FMIImport at /path/to/FMIImport/snZaf/src/FMI2_md.jl:1055 overwritten at /path/to/FMIImport/snZaf/src/FMI2_md.jl:1068.
  ** incremental compilation may be fatally broken for this module **
```

**Comment by ThummeTo** [comment]
Jap, will be fixed in the next release, thank you!

**Comment by ThummeTo** [comment]
Fixed in the current release v0.11.0
